src/Controller/EmbargoesNodeEmbargoesController.php

```diff
@@ -62,7 +63,7 @@ public function showEmbargoes($node = NULL) {
           'exempt_ips' => $ip_range_formatted,
           'exempt_users' => $formatted_exempt_users_row,
           'additional_emails' => $formatted_emails,
-          'edit' => Markup::create("<a href='/node/{$node}/embargoes/{$embargo_id}'>Edit</a><br><a href='/admin/config/content/embargoes/settings/embargoes/{$embargo_id}/delete'>Delete</a>"),
+          'edit' => Markup::create("<a href='/node/{$node->id()}/embargoes/{$embargo_id}'>Edit</a><br><a href='/admin/config/content/embargoes/settings/embargoes/{$embargo_id}/delete'>Delete</a>"),
```

Comment:
Again, more related to the idea of a bigger hardening pass (and also applicable below), but... these probably shouldn't be hardcoding links like this... probably something of an issue in the context of multisites and language handling.